.github/workflows/build-desktop.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             - name: Set up Node.js
               uses: actions/setup-node@v1
               with:
-                  node-version: 16.4.2
+                  node-version: 18
```

Comment:
Does this mean it'll always use latest version, 18.0.0, etc.? I'd prefer to be explicit with the version here.